packager/media/formats/mp2t/mpeg1_header.h

```diff
@@ -0,0 +1,54 @@
+#ifndef PACKAGER_MEDIA_FORMATS_MP2T_MPEG1_HEADER_H_
```

Comment:
Can you add a unit test for mpeg1_header? You can reference how we do it with Ac3: https://github.com/google/shaka-packager/blob/6b4a75b6bacdac6b0b9f598c365c8f84f0a7fd70/packager/media/formats/mp2t/ac3_header_unittest.cc